client/Components/Reservation.css

```diff
@@ -1,19 +1,18 @@
-html {
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+*, *:before, *:after {
+  box-sizing: border-box;
+  font-family: Circular, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
 }
 
-/* html {
-  display: grid;
-  grid-template-columns: 2fr 1fr;
-  grid-template-rows: repeat(5, 1fr);
-} */
+.positioner {
 
-*, *:before, *:after {
-  box-sizing: border-box;
 }
 
 .wrapper {
+  position: absolute;
+  top: 24px;
+  margin-left: 45px;
   display: flex;
+  --page-shell-max-content-width: 1440px;
```

Comment:
This looks interesting. I'll be reaching out to see what this is doing.

Reply:
It's an attempt at containing the edge of my module. So far it is not working.

Reply:
Ah. The "--" looks like it's custom css. Did max-width or width not achieve what you wanted?